docker-compose.yml

```
@@ -34,6 +34,14 @@ services:
             AWS_ACCESS_KEY_ID: ${S3_ACCESS_KEY_ID}
             AWS_SECRET_ACCESS_KEY: ${S3_SECRET_ACCESS_KEY}
 
+    jaeger:
+        image: jaegertracing/all-in-one:1.42
```

```suggestion
        image: jaegertracing/all-in-one:1
```

if compatible, as we are also using postgres:12